### PR TITLE
Added saving / loading of branding data

### PR DESCRIPTION
**Pull request by fordacious**

### CODE REVIEW

File: client/src/screens/AccountBranding.js [hunk 1]
```diff
@@ -17,63 +18,58 @@ var defaultExposureText = `Next Steps:
...
-  const [diagnosisText, setDiagnosisText] = useState(getDiagnosisText());
-  const [exposureText, setExposureText] = useState(getExposureText());
+  var localDiagnosisText = store.organization ? store.organization.diagnosisText : defaultDiagnosisText;
+  var localExposureText = store.organization ? store.organization.exposureText : defaultExposureText;
+
+  const [diagnosisText, setDiagnosisText] = useState(localDiagnosisText);
+  const [exposureText, setExposureText] = useState(localExposureText);
 
   const saveAccountBrandingData = () => {
     console.log("TODO save account branding data");
```
Comment (ibeckermayer) [inline client/src/screens/AccountBranding.js:29]: This func can be deleted now

File: client/src/screens/AccountBranding.js [hunk 2]
```diff
@@ -17,63 +18,58 @@ var defaultExposureText = `Next Steps:
...
+
   return (
     <div className="module-container">
       <h1 className="branding-header">Account Branding</h1>
       <div className="branding-container">
         <div className="branding-section">
           <h2 className="section-heading">Share Positive Diagnosis</h2>
           <p className="section-description">
-          This text wil be displayed ot anyone who shares a positive diagnosis and notifies everyone.
+          This text will be displayed to anyone who shares a positive diagnosis and notifies everyone.
```
Comment (ibeckermayer) [inline client/src/screens/AccountBranding.js:54]: Good catch

File: client/src/store/index.js [hunk 3]
```diff
@@ -56,9 +56,19 @@ const Store = types
       }
     })
 
+    const setOrganizationalBranding = flow(function* (diagnosisMessage, exposureMessage) {
```
Comment (ibeckermayer) [inline client/src/store/index.js:59]: This properties should be saved to the db. Hit me up in Slack and I can show you how.

Comment (fordacious) [reply client/src/store/index.js:59]: ah I assumed there was a hidden setter for the properties.

**Comment by therealnikhil** [comment]
If you login, and try refreshing manage teams, settings or any screen, it will no longer redirect you to login again, and will instead reload data from the db and show you the screen

**Pull request merged**